avro/README.md

```diff
@@ -111,6 +111,36 @@ byte[] avroData = mapper.writer(schema)
 
 and that's about it, for now.
 
+## Java Time Support
+Serialization and deserialization support for limited set of `java.time` classes to Avro with [logical type](http://avro.apache.org/docs/current/spec.html#Logical+Types) is provided by `AvroJavaTimeModule`.
+
```

Comment:
Another thing I realized: this should perhaps mention that this module is to be used either:

1. Instead of Java 8 date/time module or
2. to override Java 8 date/time module

and that for (2) module must be registered AFTER java 8 date/time module (last registration wins).

Reply:
Added here and also to AvroJavaTimeModule.